### PR TITLE
Update to serverless docs with links to new sample apps

### PR DESCRIPTION
**Pull request by jeastham1993**
### What does this PR do? What is the motivation?

The getting started pages for AWS Lambda linked to an old version of the serverless-sample-app repository. The instructions and screenshots no longer matched the application. This PR updates the docs, as well as adding links to the repo for the runtime specific install pages.

### Merge instructions

Should have a review by a member of the serverless engineering team.

Merge queue is enabled in this repo. To have it automatically merged after it receives the required reviews, create the PR (from a branch that follows the `<yourname>/description` naming convention) and then add the following PR comment:

```
/merge
```

### CODE REVIEW

**Comment by drichards-87** [comment]
Created DOCS-9377 for Docs Team editorial review.

**Comment by aliciascott** [comment]
Thank you! Created [Editorial review](https://datadoghq.atlassian.net/browse/DOCS-9451) card

File: content/en/getting_started/serverless/_index.md [hunk 1]
```diff
@@ -104,21 +68,21 @@ Each invocation is associated with a trace. Click on **Open Trace** to see the t
...
+The **Flame Graph** tab shows exactly what happened during the duration of this invocation, including which services had the highest percentage of the total execution time. The flame graph displays the request as it travels from APIGateway, through your `create-product-function`. 
+
+If you zoom out, you will also see the entire end to end trace through all of the downstream services.
 
 {{< img src="getting_started/serverless/trace_map.png" alt="Close-up of two functions" style="width:80%;">}}
 
 The **Trace Map** tab visualizes the flow of your services and how they connect to each other.
 
-The lower half of the detailed trace view displays a stack trace, which reveals the line of code responsible for throwing the error:
+If you are viewing a trace with an error the lower half of the detailed trace view will show the details:
```
Comment (cswatt) [inline content/en/getting_started/serverless/_index.md:79]:
```suggestion
If you are viewing a trace with an error, the lower half of the detailed trace view shows the details:
```

**Review by cswatt — CHANGES_REQUESTED** [review]
Thank you so much for this! I've left some smaller grammar nits.

However, instead of replacing images, could you:
- Keep the old images
- Add the new images under a new filename (e.g. `trace_map_2.png`)
- Update the img tags to reference the new filename

My team will clean up the old images after changes have been propagated across translations. Thank you!

**Comment by jeastham1993** [comment]
@cswatt sorry for the delay on this, I've just been through and added all your suggestions + update the images. This can be merged whenever you are happy!

**Pull request merged**